qa/src/test/java/io/camunda/zeebe/process/test/qa/util/Utilities.java

```diff
@@ -149,15 +150,16 @@ public static PublishMessageResponse sendMessage(
     return response;
   }
 
-  public static void increaseTime(final InMemoryEngine engine, final Duration duration) {
+  public static void increaseTime(final InMemoryEngine engine, final Duration duration)
+      throws InterruptedException {
     engine.increaseTime(duration);
     try {
-      // we need to wait some physical time so that InMemoryEngine has a chance to fire the timers
-      Thread.sleep(100);
-    } catch (InterruptedException e) {
-      // do nothing
+      engine.waitForProcessingState(Duration.ofSeconds(1));
```

Comment:
:wrench: maybe also add a `waitForProcessingState` method to this class so the two lines look more alike